pkg/config/install_requirements.go

```diff
@@ -525,6 +525,10 @@ func LoadRequirementsConfig(dir string) (*RequirementsConfig, string, error) {
 		if err == nil {
 			return config, fileName, nil
 		}
+		_, isPathErr := err.(*os.PathError)
```

Comment:
Not sure whether an explicit `util.FileExists` would be better here. It might mean that we check for existence twice, but I don't think that matters. This is really now depending on an implementation detail and there is really no guarantee that `LoadRequirementsConfigFile` is going to pass up the `PathError`. It could for example wrap the error.

Reply:
I think `errors.Wrap` is used. We need to `errors.Unwrap`.

Reply:
Fixed